src/V3Undriven.cpp

```diff
@@ -381,7 +390,21 @@ class UndrivenVisitor final : public VNVisitor {
                     UINFO(9, " Full bus.  Entryp=" << cvtToHex(entryp) << endl);
                     warnAlwCombOrder(nodep);
                 }
+                if (m_alwaysCombp && entryp->isDrivenWhole() && !entryp->isDrivenAlwaysCombWhole()) {
+                    nodep->v3warn(MULTIDRIVEN,
+                                  "Variable written to in always_comb also written by other process"
+                                      << " (IEEE 1800-2017 9.2.2.2): "
+                                      << nodep->prettyNameQ());
+                }
+                if (!m_alwaysCombp && entryp->isDrivenWhole() && entryp->isDrivenAlwaysCombWhole()) {
+                    nodep->v3warn(MULTIDRIVEN,
+                                  "Variable also written to in always_comb"
+                                      << " (IEEE 1800-2017 9.2.2.2): "
+                                      << nodep->prettyNameQ());
```

Comment:
Ideally we'd also print the other location, similar to this warning:

 ```
                   "Unsupported: Blocked and non-blocking assignments to same variable: "
                        << vscp->varp()->prettyNameQ() << '\n'
                        << vscp->warnContextPrimary() << '\n'
                        << blockingp->warnOther() << "... Location of blocking assignment\n"
                        << blockingp->warnContextSecondary() << '\n'
                        << nonblockingp->warnOther() << "... Location of nonblocking assignment\n"
                        << nonblockingp->warnContextSecondary());
 ```
 
 To do this save the varref of the last blocking/nonblocking assign.